Moderation-Policy.md

```diff
@@ -0,0 +1,113 @@
+
+### Applicability
+
+By default, this policy applies to all repositories under the Node.js Github Organization and all Node.js Working Groups.
+
+Individual Working Groups and Top Level Projects chartered by the TSC may adopt an alternative Moderation Policy for any repository under their stewardship so long as:
+* The Moderation Policy is openly documented as part of the Working Group or Project Charter and;
+* Includes provisions for clearly and openly documenting Moderation actions taken.
+
+If a particular repository can be considered to fall under the stewardship of multiple Working Groups (or Top Level Projects) that have adopted different Moderation Policies, they can choose to either:
+* Decide between themselves which Moderation Policy will be in effect for the repository in question, or
+* Ask the TSC to determine which Moderation Policy should apply.
+
+Any alternative Moderation Policy used for a given repository must be included in the root directory of the repository using the `Moderation-Policy.md` filename. If a repository does not contain a `Moderation-Policy.md` file, then this default policy applies.
+
+### Terms
+
+* "Collaborator" refers to any individual member or "outside collaborator" of the Node.js Github Organization.
+* "Post" refers to the content of any issue, pull request, comment or wiki page.
+* "Moderate" refers to the act of modifying the content of or deleting any Post.
+* "Ban" refers to the act of blocking an individual Github account from any further participation in the Node.js Github Organization.
+
+### Requesting Moderation
+
+Anyone may request Moderation of a Post.
+
+Moderation requests are only necessary when:
+ 
+* The Post in question was authored by a Collaborator, or
+* The request is originating from a non-Collaborator.
+
+Requesting Moderation of a Post can be accomplished in one of five ways:
+
+* Via the `report@node.js` email address,
+* Via private email to individual TSC members,
+* Via a new Post in the same thread as the Post being requested for Moderation,
+* Via a new Post in the private nodejs/moderation repository, or
+* Private communication between the individuals involved,
+
+Use of the `report@node.js` email address, or private email to individual TSC members, is appropriate when the individual requesting the Moderation does not feel comfortable directly or publicly making the request. All emails sent to the `report@node.js` address are currently forwarded to all members of the TSC. [[ED NOTE: This has not yet been set up]]
+
+Requests should contain as much information and context as possible, including the URL and a screenshot of the Post in question. Screenshots may be modified to obscure obscene or offensive content.
+
+External public venues or social media services such as Twitter should never be used to request Moderation or to discuss the details of a Moderation request.
+
+Multiple moderation requests for the same Post are unnecessary.
```

Comment:
To me not sending multiple requests is a common sense kind of thing. The individual who will abuse this will do so whether or not this is in the document.